docs/reference/eql/search.asciidoc

```diff
@@ -74,7 +75,8 @@ The API returns the following response containing the matching event:
             "name": "cmd.exe",
             "path": "C:\\Windows\\System32\\cmd.exe"
           }
-        }
+        },
+        "sort" : [1607339167000]
```

Comment:
@jrodewig Not sure whether there's a better way to match the document - essentially the score has been replaced by sort and I'd like to remove both from the result.

Reply:
@costin  No worries. As long as the CI tests pass, I would not consider this a blocker. I can work on the additional doc changes as a separate PR.

Reply:
Related PR: #53027 53027